apps/support/Makefile.inc

```diff
@@ -6,7 +6,7 @@ ifeq ($(OS), Windows_NT)
 # Flags for mingw environment
 LDFLAGS += -lole32 -luuid -Wl,--stack,8388608
 else
-LDFLAGS += -ldl -lpthread -lz
+LDFLAGS += -ldl -lpthread -lz -ltinfo
```

Comment:
I believe this is adding one of the terminal info libraries to the build. I would really prefer not to have this happen. I configure LLVM with all the terminfo stuff off by default. ("-DLLVM_ENABLE_TERMINFO=OFF" to cmake.) Frankly defaulting this to on in the LLVM build is a mistake, but I definitely do not want the dependency creeping into Halide. If it is a dep for llvm, it should somehow show up in llvm-config and end up getting linked that way. If that does not work, perhaps a bug against LLVM is in order. I also don't think this library is there by default on Mac OS X.